src/settings/signals.py

```
@@ -0,0 +1,39 @@
+import logging
+import os
+from django.conf import settings
```

Ruff

`django.conf.settings` imported but unused

Remove unused import: `django.conf.settings`

(F401)